CRM/Core/DAO.php

```diff
@@ -186,16 +186,6 @@ public static function init($dsn) {
     }
     $factory = new CRM_Contact_DAO_Factory();
     CRM_Core_DAO::setFactory($factory);
-    $currentModes = CRM_Utils_SQL::getSqlModes();
-    if (CRM_Utils_Constant::value('CIVICRM_MYSQL_STRICT', CRM_Utils_System::isDevelopment())) {
-      if (CRM_Utils_SQL::supportsFullGroupBy() && !in_array('ONLY_FULL_GROUP_BY', $currentModes) && CRM_Utils_SQL::isGroupByModeInDefault()) {
-        $currentModes[] = 'ONLY_FULL_GROUP_BY';
-      }
-      if (!in_array('STRICT_TRANS_TABLES', $currentModes)) {
```

Comment:
@eileenmcnaughton @demeritcowboy I just checked and on the MySQL5.6 STRICT_TRANS_TABLES is not in the default sql_mode but it is in MySQL 5.7, MySQL 5.7 is what PR tests and the max matrix tests run against, min is run against MySQL 5.6

Reply:
Is there a way to set STRICT_TRANS_TABLES and ONLY_FULL_GROUP_BY as part of jenkins config?

Reply:
@seamuslee001 yeah - I think the point is we have enough test / dev cover on it now without this